<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSpinBox" name="size_of_system">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>210</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_size">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ромірність системи</string>
    </property>
   </widget>
   <widget class="QComboBox" name="chooseMethod">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_choose">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Метод рішення</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGetSolution">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Знайти рішення</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGetRandom">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>420</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Задати випадково</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGetLog">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>420</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Продивитись журнал</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClearField">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Очистити</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_a">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Матриця системи</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_b">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>10</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Вільні члени</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_solution">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>10</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Рішення</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableA">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>421</width>
      <height>331</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>3</number>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>45</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>80</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <row/>
    <row/>
    <row/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QTableWidget" name="tableB">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>50</y>
      <width>151</width>
      <height>331</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>3</number>
    </property>
    <property name="columnCount">
     <number>1</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>45</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>120</number>
    </attribute>
    <row/>
    <row/>
    <row/>
    <column/>
   </widget>
   <widget class="QTableWidget" name="tableX">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>50</y>
      <width>151</width>
      <height>331</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="rowCount">
     <number>3</number>
    </property>
    <property name="columnCount">
     <number>1</number>
    </property>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>45</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>120</number>
    </attribute>
    <row/>
    <row/>
    <row/>
    <column/>
   </widget>
   <widget class="QLabel" name="label_size_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Точність рішення</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_accuracy">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>300</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0.05</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1040</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>Зберегти рішення</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Відкрити систему</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
